projects/igniteui-angular/src/lib/grids/filtering/grid-filtering-row.component.ts

```diff
@@ -753,7 +753,8 @@ export class IgxGridFilteringRowComponent implements AfterViewInit {
         const containerRect = this.container.nativeElement.getBoundingClientRect();
 
         for (let index = 0; index < chipAraeChildren.length; index++) {
-            if (Math.ceil(chipAraeChildren[index].getBoundingClientRect().left) < Math.ceil(containerRect.left)) {
+            if (Math.ceil(chipAraeChildren[index].getBoundingClientRect().left) < Math.ceil(containerRect.left) &&
```

Comment:
The first condition seems unnecessary. If the second is true, the first one will always be true.

Reply:
Done